### DISCUSSION TITLE
Vue.js 3.x tsParticles component

### DISCUSSION THREAD

**Discussion by matteobruni** [matteobruni/tsparticles#1129] [General]
This section will be used to talk about the `Vue.js 3.x` version of tsParticles, the `particles.vue3` package.

**Comment by baizhizhen** [comment]
if use vue3 create a project, and then import particles.vue3 and used, that has no result

**Reply by baizhizhen** [reply]
vue-cli

**Reply by matteobruni** [reply]
I need more details about it, can you share some steps to reproduce the issue?

**Comment by matteobruni** [comment]
This discussion will be closed since there's a dedicated repository for the Vue 3.x component: https://github.com/tsparticles/vue3
